code/edgebridge/build.gradle

```diff
@@ -46,7 +45,6 @@ android {
     buildTypes {
         debug {
             testCoverageEnabled true
-            debuggable true
```

Comment:
Can you keep this parameter?  I realize the AGP Migration Assistant removes it stating it is ignored, but I've also read documentation that it is needed to retain the debugging symbols of the library. To be safe, let's keep it in the build configuration.